src/app/pages/psbt-request/components/psbt-decoded-request/psbt-decoded-request-node/psbt-unsigned-input-list/components/psbt-unsigned-input-item.tsx

```diff
@@ -12,33 +14,34 @@ import { PsbtUnsignedInputWithInscription } from './psbt-unsigned-input-with-ins
 interface PsbtUnsignedInputItemProps {
   addressNativeSegwit: string;
   addressTaproot: string;
-  utxo: TaprootUtxo & OrdApiInscriptionTxOutput;
+  utxo: BitcoinTransactionVectorOutput;
+  utxoWithInscription: (TaprootUtxo & OrdApiInscriptionTxOutput) | undefined;
```

Comment:
Isn't this passing duplicate data, the UTXO twice? Can't we just pass the inscription data separately?

```suggestion
  utxoWithInscription?: TaprootUtxo & OrdApiInscriptionTxOutput;
```